src/JsonSchema/Pointer.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace JsonSchema;
+
+use JsonSchema\Exception\InvalidPointerException;
+use JsonSchema\Exception\ResourceNotFoundException;
+
+/**
+ * Resolve JSON Pointers (RFC 6901)
+ */
+class Pointer
```

Comment:
Very good point, in putting the tests together I've identified a number of bugs!